fic_modules/git.py

```diff
@@ -489,33 +505,38 @@ def filter_git_commit_keyword(repository_name, repository_path):
     last_checked = last_check(repository_name)
     new_commit_dict = {"0": {"lastChecked": str(datetime.utcnow())}}
     if limit_checker():
-        new_commits = GIT.get_repo(repository_path).get_commits(since=last_checked)
+        new_commits = GIT \
+            .get_repo(repository_path) \
+            .get_commits(since=last_checked)
     for commit in new_commits:
         files_changed_by_commit = [x.filename for x in commit.files]
         if files_changed_by_commit:
             each_commit = {}
             if LOGGER:
-                print(commit.commit.message)
+                LOGGER.info(commit.commit.message)
```

Comment:
Should not we remove `if LOGGER`? (line 515)
As logging will be handled at config level, not arguments.
For example: `logging.basicConfig(level=logging.INFO)`

Reply:
Fixed in the next commit.